tenzir/integration/tests/tcp.bats

```diff
@@ -1,4 +1,4 @@
-: "${BATS_TEST_TIMEOUT:=3}"
+: "${BATS_TEST_TIMEOUT:=10}"
```

Comment:
Is there a good reason why we specify a test timeout for this suite that separate from the globally configured one in the first place?

Reply:
The test sometimes ran into the 3 second timeout when they are run in parallel. I don't see a problem in upping the threshold a bit for that.

Reply:
My suggestion was to axe the entire line and to just use the default instead of setting a custom one in the first place, assuming a default value exists.

Reply:
In that case the timeout for every test would be dictated by the one test with the highest expected runtime.
I believe setting per-file thresholds provides a better testing UX.

Reply:
Makes sense. I don't have too strong of an opinion on this, so let's just keep it as-is then.